### PR TITLE
added links/data to python and added java,c++ to the languages sectio…

### PR DESCRIPTION
**Pull request by anonymgit**
…n with the resources link

## Fixes Issue
- issue #461 ,want to add some basic languages and it resources

## Changes proposed
- changes made in database>languages>cplusplus.json/java.json/python.json
- all the changes are tested on local machine

### previous
image
### now
image

## Note to reviewers
- I have added some basic languages and their public resources to the database. Kindly evaluate the PR and give a feedback about that. I hope you like a small contribution from my side to this repo.

### CODE REVIEW

File: database/languages/cplusplus.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "name": "c++ official",
+        "description": "It is an official website of cplusplus where you can get tutorial that provides detailed explanation, examples, and tasks of c++ language.",
```
Comment (CBID2) [inline database/languages/cplusplus.json:4]:
```suggestion
        "description": "This is C++'s official website.",
```

File: database/languages/cplusplus.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "url": "https://cplusplus.com/doc/tutorial/",
+        "category": "Other",
+        "subcategory": "cplusplus"
+    },
+    {
+        "name": "c++ tutorialspoint ",
+        "description": "It is an website of cplusplus where you can get tutorial that provides detailed explanation, examples, and tasks of c++ language.",
+        "url": "https://www.tutorialspoint.com/cplusplus/index.htm",
+        "category": "Other",
+        "subcategory": "cplusplus"
```
Comment (CBID2) [inline database/languages/cplusplus.json:14]: Hmmm... 🤔, is this a tutorial on how to use C++ or a project where knowledge of this language is required?  If it's the former, I highly recommend replacing this with a source that describes my second idea.

File: database/languages/cplusplus.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+        "name": "c++ w3School",
+        "description": "Here you will get a detailed info about c++ concept with integrated online IDE where you can play around the code and practice while learning .",
```
Comment (CBID2) [inline database/languages/cplusplus.json:18]:
```suggestion
        "name": "c++ w3School",
        "description": "Here, you will learn C++ by doing coding exercises alongside the tutorial  in an integrated online IDE.",
```

File: database/languages/cplusplus.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        "url": "https://www.w3schools.com/cpp/default.asp",
+        "subcategory": "cplusplus"
+    },
+    {
+        "name": "c++ javatpoint",
+        "description": "This resource contains a detailed theoretical and example of c++ of each and every concept of c++,you will get source code along with the tutorials.",
```
Comment (CBID2) [inline database/languages/cplusplus.json:24]:
```suggestion
        "description": "This resource contains a detailed theory and examples of C++. Also, you will get source code along with the tutorials.",
```

File: database/languages/java.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "name": "Java official",
+        "description": "It is an official website that provides detailed explanation, examples, and tasks to learn Java,you can find detailed info about java inbuilt classes and its concepts.",
```
Comment (CBID2) [inline database/languages/java.json:4]:
```suggestion
        "description": "This is Java's official website.  There, you will be provided with detailed explanations, examples, and tasks. You will also be provided with detailed info about Java inbuilt classes and its concepts.",
```

File: database/languages/java.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        "url": "https://docs.oracle.com/javase/tutorial/",
+        "category": "Other",
+        "subcategory": "java"
+    },
+    {
+        "name": "Java - javatpoint",
+        "description": "This resource contains a detailed theoretical and practical and example of each and every concept of java,you will get source code along with the tutorials.",
```
Comment (CBID2) [inline database/languages/java.json:11]:
```suggestion
        "description": "This resource contains detailed theories and practical examples of each concept in Java. Also. you will get source code with the tutorials.",
```

File: database/languages/java.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        "url": "https://www.javatpoint.com/java-tutorial",
+        "category": "Other",
+        "subcategory": "java"
+    },
+    {
+        "name": "Java - w3School",
+        "description": "Here you will get a detailed info about java concept with integrated online IDE where you can play around the code and practice while learning .",
```
Comment (CBID2) [inline database/languages/java.json:18]:
```suggestion
        "name": "Java - w3School",
        "description": "Here, you get to practice and learn Java by doing exercises alongside a tutorial  via an integrated online IDE.",
```

File: database/languages/java.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        "url": "https://www.w3schools.com/java/",
+        "category": "Other",
+        "subcategory": "java"
+    },
+    {
+        "name": "java- tutorialspoint ",
+        "description": "It is an website of java where you can get tutorial that provides detailed explanation, examples, and tasks of java language.",
```
Comment (CBID2) [inline database/languages/java.json:25]:
```suggestion
        "name": "java- tutorialspoint ",
        "description": "Here, you get tutorials that provide detailed explanations, and examples of using Java.",
```

File: database/languages/python.json [hunk 9]
```diff
@@ -1,3 +1,30 @@
...
+    {
+        "name": "Python official",
+        "description": "It is a official python docs to learn python.",
+        "url": "https://docs.python.org/3/",
+        "category": "Other",
+        "subcategory": "python"
+    },
+    {
+        "name": "Python - GFG",
+        "description": "It has a detailed theoretical and practical resources to learn python. You will get step by step guide from installation to compilation of the source code,you can practice while learning on this platform.",
```
Comment (CBID2) [inline database/languages/python.json:11]:
```suggestion
        "description": "It has detailed resources to learn Python. You will get a step-by-step guide on how to install the code and complete the practice exercises.",
```

File: database/languages/python.json [hunk 10]
```diff
@@ -1,3 +1,30 @@
...
+        "url": "https://www.geeksforgeeks.org/python-programming-language/learn-python-tutorial/",
+        "category": "Other",
+        "subcategory": "python"
+    },
+    {
+        "name": "Python - realPython",
+        "description": "This page contains a beginner to advance python learning path with best examples and it also helps for pyhton developers to understand the in built libraries.",
```
Comment (CBID2) [inline database/languages/python.json:18]:
```suggestion
        "name": "Python - realPython",
        "description": "This page contains a beginner to advance learning path and helps Python developers to learn how to use the in-built libraries.",
```

File: database/languages/python.json [hunk 11]
```diff
@@ -1,3 +1,30 @@
...
+        "url": "https://realpython.com/",
+        "category": "Other",
+        "subcategory": "python"
+    },
+    {
+        "name": "Python-w3School",
+        "description": "It has beginner friendly ,well structured python tutorials where you can practice while learning python and you can also practice exercise available on the platform.",
```
Comment (CBID2) [inline database/languages/python.json:25]:
```suggestion
        "description": "It has beginner-friendly, well-structured Python tutorials where you can practice while learning. You can also practice the exercises on the platform.",
```

File: database/languages/java.json [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+        "url": "https://www.tutorialspoint.com/java/index.htm",
+        "category": "Other",
+        "subcategory": "java"
+    },
+    {
+        "name": "java- GFG ",
+        "description": "This website contains a detailed theoretical and practical knowledge anout java ,it has an integrated online IDE where you can practice while learning java ",
```
Comment (CBID2) [inline database/languages/java.json:32]:
```suggestion
        "name": "java- GFG ",
        "description": "This website contains detailed and practical information about Java. It also has an integrated online IDE where you can practice the exercises while viewing the tutorials",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @anonymgit! :) I just made some suggestions on how you can make your descriptions clearer. Also, please push your PR to a different branch. Check [step 11](https://github.com/rupali-codes/LinksHub#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines) on the Contributing file to learn how. Keep it up! :) 👍🏾

**Comment by anonymgit** [comment]
> Hey @anonymgit! :) I just made some suggestions on how you can make your descriptions clearer. Also, please push your PR to a different branch. Check [step 11](https://github.com/rupali-codes/LinksHub#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines) on the Contributing file to learn how. Keep it up! :) 👍🏾

> Thank you so much for your suggestions and guidance! I appreciate your feedback and will definitely work on making the descriptions clearer. I will also make sure to push my PR to a different branch, as per the guidelines mentioned in step 11. Your support means a lot to me, thanks again! 👍🏾

**Comment by anonymgit** [comment]
> Hey @anonymgit! :) I just made some suggestions on how you can make your descriptions clearer. Also, please push your PR to a different branch. Check [step 11](https://github.com/rupali-codes/LinksHub#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines) on the Contributing file to learn how. Furthermore, resolve the conflicts. Keep it up! :) 👍🏾

> Hey @CBID2 , created a PR according to  [step 11](https://github.com/rupali-codes/LinksHub#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines), should I close this PR ?

**Comment by CBID2** [comment]
> > Hey @anonymgit! :) I just made some suggestions on how you can make your descriptions clearer. Also, please push your PR to a different branch. Check [step 11](https://github.com/rupali-codes/LinksHub#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines) on the Contributing file to learn how. Furthermore, resolve the conflicts. Keep it up! :) 👍🏾
> 
> 
> 
> > Hey @CBID2 , created a PR according to  [step 11](https://github.com/rupali-codes/LinksHub#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines), should I close this PR ?

Sure

**Pull request closed without merging**